sources/install.sh

```diff
@@ -3228,6 +3228,13 @@ function install_PassTheCert() {
   add-test-command "passthecert.py --help"
 }
 
+function install_haiti() {
+  colorecho "Installing haiti"
+  gem install haiti-hash
+  add-history haiti
```

Comment:
The `add-history` function will look for the `haiti` file in the `sources/history.d/` directory but it seems your pull request doesn't include that file. Please add the `haiti` history file containing the history line(s) to the proper directory.

Reply:
I was wondering what `add-history` was for but found no documentation about it. So I should put some command examples in `sources/zsh/history.d/haiti`?

Reply:
Indeed, contributors documentation is yet to be written. The function is defined here: https://github.com/ThePorgs/Exegol-images/blob/dev/sources/install.sh#L30-L34

But yes, you need to put one or multiple command history in a `sources/zsh/history.d/haiti` file